### PR TITLE
Initial Draft of Hindi Translation

### PR DESCRIPTION
**Pull request by MrSourabh**
Hi, This is initial draft with partial translation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ I'm on the journey, too. Follow along on my blog at [GoogleyAsHeck.com](https://
 
 ## About Google
 
-- [ ] For students - [Google Careers: Technical Development Guide](https://www.google.com/about/careers/students/guide-to-technical-development.html)
+- [x] For students - [Google Careers: Technical Development Guide](https://www.google.com/about/careers/students/guide-to-technical-development.html)
```
Comment (jwasham) [inline README.md]: Please remove the x here.

**Comment by jwasham** [comment]
Did you want me to pull this now, or once it's done?

**Comment by MrSourabh** [comment]
Let's merge it once it is completed.

**Comment by micthawatchai** [comment]
ถ้าดึงคืนไม่ได้ฉะนพอจะมีวิธีแก้ไขหน้าเว็บที่หายไปได้ไหมครับ

**Comment by MrSourabh** [comment]
@jwasham  Can you please merge this, this is not complete, but might help someone who is  interested in get going

File: README.md (hunk 1 above)
Comment (MrSourabh) [inline README.md]: Done

**Pull request merged**